src/mbgl/layout/symbol_instance.hpp

```diff
@@ -7,17 +7,27 @@
 
 namespace mbgl {
 
-struct Anchor;
```

Comment:
`struct` and `class` are interchangeable, modulo default visibility, but annoyingly, the keyword used for forward declaration must match the keyword used for the definition. Our convention is to always use `class`, so that forward declarations are consistent.